vite.config.js

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+//import nodePolyfills from 'rollup-plugin-node-polyfills';
+
+export default defineConfig(() => {
+  return {
+    resolve: {
+      alias: {
+        'util': 'util',
+      }
+    },
+    server: {
+      port: 3000,
+    },
+    build: {
+      outDir: 'build',
+    },
+    plugins: [react()],
+    define: {
+      'process.env.NODE_DEBUG': JSON.stringify(process.env.NODE_DEBUG),
+    },
```

Comment:
> The whole config overriding thing during the build process ./src/lib/babel-replace-config-import.js

Since this is only changing the imported file I think we could just as easily get away with skipping the import and using a `define` here instead. We can probably eliminate the custom Babel loader and instead use this inline here:

```js
define: {
	CONFIG_MODULE: process.env.WPCOM_BUILD ? '../config.wpcom' : '../config',
}
```

```js
import config from CONFIG_MODULE;
```

I'm not sure if we can do this but it wouldn't take much to transform that into a define we can directly use without importing; as in JSON-serializing the config value and then directly reading it inside the `src/spi/index.js` file

```js
const config = JSON.parse(CONFIG);
```

Reply:
Brilliant! I got it to work by defining the json string of the config.